app/html/index.html

```diff
@@ -16,7 +34,11 @@
     <p id="loginStatus">&nbsp;</p>
   </div>
 
-  <script src="js/index.js"></script>
```

Comment:
This script should be added back in, it handles login/logout.

Reply:
In fact, I didn't deleted it, but just moved it to the top side. I meant to bring the importing the code for javascript together, but I think it's right to put it back there if you need it here.

Reply:
ahhhh ok I missed that, disregard this then :)